### DISCUSSION TITLE
Evaluate file issue: The export {0} was exported multiple times

### DISCUSSION THREAD

**Discussion by tamaramironov** [microsoft/vscode-powerquery-sdk#269] [Q&A] [answered]
Hello! We have recently migrated from VS 2019 to using VS code SDK. Our connectors are buildable and work as expected. However when I try to use "Evaluate current file" feature I get this error for which I can't find any documentation. 
 
```
[5:57:22 PM]	[Info]	RunTestBattery result [
    {
        "ActivityId": "c687e075-c9e1-405c-bfcb-8198a222d986",
        "DataSourceAnalysis": [
            {
                "Kind": "UnknownFunction",
                "FunctionName": "Extension.LoadString"
            },
            {
                "Kind": "UnknownCallSite",
                "FunctionName": "Web.Contents"
            },
            {
                "Kind": "UnknownFunction",
                "FunctionName": "Extension.Contents"
            }
        ],
        "Details": "The export XYZ_Reporting was exported multiple times.",
        "EndTime": "2023-04-24T17:57:22.5595115+00:00",
        "Method": "XYZ_Reporting",
        "Name": "XYZReporting.pq",
        "StartTime": "2023-04-24T17:57:21.3365238+00:00",
        "RowCount": 0,
        "Status": "Failed",
        "Type": "PQTest.RunTest",
        "Error": {
            "Message": "The export XYZ_Reporting was exported multiple times.",
            "MessageFormat": "The export {0} was exported multiple times.",
            "MessageParameters": [
                "XYZ_Reporting"
            ],
            "Details": {
                "Reason": "Expression.Error"
            }
        }
    }
]
```
It doesn't seem like anything is exported multiple times. Connector works fine and can be tested manually, however this blocks using SDK features. I can't share the source code but want to find out more about this error.

**Comment by geologyrocks** [comment, marked as answer]
Hey there,

We also recently migrated to VS Code from Visual Studio 2019.  Not sure on your build setup, but you need to run Evaluate current file in the `ConnectorName.query.pq` file and not the `ConnectorName.pq` file.

Alternatively, if you have a flat folder structure with everything in the connector folder, you can press `Ctrl+Shift+B` and select `Make with PQX`; it will build your connector every time.

Hope that helps,

geologyrocks

**Reply by tamaramironov** [reply]
Thank you @geologyrocks, you're right, I tried to use Evaluate current file on the "main" .pq file in my project.
